.github/workflows/minikube-image-benchmark.yml

```diff
@@ -0,0 +1,31 @@
+name: "build-and-run Publish Chart"
+on:
+  workflow_dispatch:
+  schedule:
+    # every day at 7am & 7pm pacific
+    - cron: "0 2,14 * * *"
+env:
+  GOPROXY: https://proxy.golang.org
+  GO_VERSION: '1.20.6'
+permissions:
+  contents: read
+
+jobs:
+  build-and-run-benchmark:
```

Comment:
also change this to image-benchmark, since we have other benchmarks we dont wanna confuse this with other jobs